gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts

```diff
@@ -0,0 +1,121 @@
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+import { ApiImport } from '@model/api-imports';
+
+describe('API Info Page functionality', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit('/#!/environments/default/apis/');
+    cy.url().should('include', '/apis/');
+  });
+
+  before(() => {
+    cy.log('Import (create) v2 API');
+    cy.request({
+      method: 'POST',
+      url: `${Cypress.env('managementApi')}${Cypress.env('defaultOrgEnv')}/apis/import`,
+      auth: { username: ADMIN_USER.username, password: ADMIN_USER.password },
+      body: ApisFaker.apiImport({ visibility: Visibility.PUBLIC }),
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      api = response.body;
+    });
+  });
+
+  const ApiDetails = new apiDetails();
+  const apiName = faker.commerce.productName();
+  const newName = apiName.replace(/ /g, '-');
+  const apiVersion = faker.datatype.number();
+  const apiDescription = faker.commerce.productDescription();
+  const path = require('path');
+  let api: ApiImport;
+  let apiId: ApiImport;
+
+  it('Verify Info Page Elements', () => {
+    cy.getByDataTestId('api_list_edit_button', { timeout: 5000 }).first().click();
+    cy.contains('Policy Studio').should('be.visible');
+    cy.contains('Info').should('be.visible');
+    cy.contains('Plans').should('be.visible');
+    ApiDetails.info().click();
+    cy.getByDataTestId('api_info_namefield').scrollIntoView().should('be.visible');
+    cy.getByDataTestId('api_info_versionfield').should('be.visible');
+    cy.getByDataTestId('api_info_descriptionfield').should('be.visible');
+    cy.getByDataTestId('api_info_labelsfield').should('be.visible');
+    cy.getByDataTestId('api_info_categoriesdropdown').should('be.visible').click();
+    cy.getByDataTestId('api_info_categorieslist-unavailable').should('be.visible');
+  });
+
+  it('Edit Info Page and Verify changes', () => {
+    cy.getByDataTestId('api_list_edit_button').first().click();
+    ApiDetails.info().click();
+    cy.getByDataTestId('api_info_namefield').scrollIntoView().clear().type(`${apiName}`);
+    cy.getByDataTestId('api_info_versionfield').clear().type(`${apiVersion}`);
+    cy.getByDataTestId('api_info_descriptionfield').clear().type(`${apiDescription}`);
+    cy.getByDataTestId('api_info_savebar', { timeout: 5000 }).contains('Save').click();
+    cy.contains('Configuration successfully saved!').should('be.visible');
+    cy.visit('/#!/environments/default/apis/');
+    cy.contains(`${apiName}`).should('be.visible');
+  });
+
+  it('Export API and verify json download', () => {
+    cy.getByDataTestId('api_list_edit_button').first().click();
+    cy.getByDataTestId('api_info_export_menu').click();
+    cy.getByDataTestId('api_info_export_api').click();
+    const downloadsFolder = Cypress.config('downloadsFolder');
+    cy.readFile(path.join(downloadsFolder, `${newName}-${apiVersion}.json`)).should('exist');
```

Comment:
same here, I would expect the content is good. but maybe it's another scenario ?

Reply:
Added `cy.readFile(path.join(downloadsFolder, `${newName}-${apiVersion}.json`)).should('contain', `${apiName}`).and('contain', `${apiDescription}`)`